lib/main.dart

```diff
@@ -31,9 +31,30 @@ Future<void> main() async {
   }
 
   if (isDesktop()) {
+    // start backend services before setting up window
     LanternFFI.startDesktopService();
     await WebsocketSubscriber().connect();
+
     await windowManager.ensureInitialized();
+    const double width = 360;
+    const double height = 712;
+
+    WindowOptions windowOptions = const WindowOptions(
+      size: ui.Size(width, height),
+      minimumSize: ui.Size(width, height),
+      maximumSize: ui.Size(width, height),
+      center: true,
+      backgroundColor: Colors.transparent,
+      skipTaskbar: false,
+      windowButtonVisibility: true,
+    );
+    await windowManager.setPreventClose(true);
+    await windowManager.setResizable(false);
+    // make sure the window is initialized before rendering the UI
+    await windowManager.waitUntilReadyToShow(windowOptions, () async {
+      await windowManager.show();
+      await windowManager.focus();
+    });
```

Comment:
I moved the window initialization back to main after I noticed there were issues where the window would randomly re-position itself on start. This makes sure the window is properly set up before rendering any widgets, and the home widget is still using a WindowListener. I added some comments for future reference.